README.md

```diff
@@ -80,6 +82,24 @@ Data lineage is just one type of metadata to help analysis and model training th
 
 <img src="https://lamin-site-assets.s3.amazonaws.com/.lamindb/6sofuDVvTANB0f480001.png" width="850">
 
+You can annotate artifacts with various ontologies. Let’s annotate the artifact with a built-in universal label ontology.
+
+```python
+# create a label
+my_experiment = ln.ULabel(name="My experiment").save()
+# annotate the artifact with the label
+artifact.ulabels.add(my_experiment)
+```
+
+Annotations help you organize your data collections and make it easier to find relevant data.
```

Comment:
"Annotations help organizing and finding artifacts. This is how you query by annotations.